src/grammar/block.rs

```diff
@@ -63,6 +64,57 @@ pub(super) fn parse_stmt(p: &mut Parser) {
     p.finish();
 }
 
+fn parse_execute_immediate_stmt(p: &mut Parser) {
+    p.start(SyntaxKind::ExecuteImmediateStmt);
+    p.expect(T![execute]);
+    p.expect(T![immediate]);
+    // Parse String
+    if !p.eat(T![unquoted_ident]) {
```

Comment:
Did you test if the variable could be quoted?

Reply:
If it is it breaks checking for statements in quotes.